src/js/player.js

```diff
@@ -1653,7 +1654,11 @@ class Player extends Component {
     // Loop through each playback technology in the options order
     for (let i = 0, j = this.options_.techOrder; i < j.length; i++) {
       let techName = toTitleCase(j[i]);
-      let tech = Component.getComponent(techName);
+      let tech = Tech.getTech(techName);
+
+      if (!tech) {
+        tech = Component.getComponent(techName);
```

Comment:
This would mean that the tech was only registered as a component? I think it's worth a comment above this to remind ourselves to one day get rid of it.